imapclient/imapclient.py

```diff
@@ -109,8 +110,6 @@ class IMAPClient(object):
     calls if required.
 
     """
-
-    Error = imaplib.IMAP4.error
```

Comment:
Maybe this should stay for backward-compatibility after all? Maybe a bit different with `Error = exceptions.IMAPClientException` but there should be no difference for people catching this in old versions.